### ISSUE TITLE
slow to open after update to 2.40.0 -- after update app crashed 

### ISSUE THREAD

**Issue by 7ekken** [PathOfBuildingCommunity/PathOfBuilding#7412]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate open **and closed** issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues?q=is%3Aissue)

### How is Path of Building expected to behave?

app should open faster than last version i would assume.  app takes about 10 seconds to open

### How does Path of Building behave?

slow to open - takes at least 10 seconds to start up.  old version before this update took roughly 1 to 2 seconds
Windows 11 Pro
AMD 5900x
32 GB RAM
Asus ROG Strix 3080oc 

### How to reproduce the issue

not sure, try it on more pc's?

**Comment by 7ekken** [comment]
to add to the above issue.  i did a complete uninstall of the app.  Downloaded the newest version and received a Windows Defender warning image below.  After installation i received the same error command prompt of missing the DLL file.

I reinstalled an older version with no internet to have functionality until bug is fixed.

![Untitled](https://github.com/PathOfBuildingCommunity/PathOfBuilding/assets/104710979/4aa5a3bf-1c3e-4f15-97df-8ca24d560556)

**Comment by zao** [comment]
On Windows SmartScreen - it loves to reject "uncommon" and "new" executables like installers. The way you allowlist the file is that you click the "More Info" link and tell it to run the program anyway.

We had some problems with the installer generation and recommend either obtaining a refreshed installer eventually or run the 2.39.x installer and let it self-update to current.

On startup time - there's a few different aspects that come into play. As it's a new program with completely new shaders, there's going to be some additional time spent in the GPU driver to generate and cache those. There's also some foundational changes in that we don't background load textures anymore, nor do we have the green splash screen with the console text that used to show early in the program startup. Some of those may have made it feel faster.

With my machine (7800X3D) it takes around 1s-1.5s before the initialization is done and 3.8s before the first two heavy frames with upfront work have completed. It's been a while since I ran it on older hardware, but I've tested it all the way down to an Athlon X2 4400+ in the past just to ensure that it could run with interactive performance.

There's not too much insight from running it on more PCs, except that "yep, we do a lot of up-front work".
